CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Changelog
 =========
 
```

Comment:
can you please add `1.3.0 (unreleased)` as title? i fear we might get confused between 2.0 and 1.3 otherwise.